.commitlintrc.js

```diff
@@ -1,9 +1,37 @@
 const typeEnum = require('@commitlint/config-angular-type-enum');
-const types = [...typeEnum.value(), 'sync', 'release'];
 
 module.exports = {
   extends: ['@commitlint/config-angular'],
```

Comment:
Also, i think we can switch to `@commitlint/config-conventional` instead as we are now not bound to follow Angular conventions.